module/extension/Director/src/Director.cpp

```diff
@@ -271,6 +271,15 @@ namespace module::extension {
                 reevaluate_group(g);
             });
 
+        on<Trigger<WaitFinished>, Sync<Director>, Pool<Director>, Priority::REALTIME>().then(
+            "Wait Delay",
+            [this](const WaitFinished& w) {
```

Comment:
Technically the same provider could start and then leave and come back. But in that case the task would have been nulled out I think.

Reply:
I feel like the task should be gone in that case